src/main_dpdk.cpp

```diff
@@ -6028,6 +6032,13 @@ COLD_FUNC int update_global_info_from_platform_file(){
         }
     }
 
+    if (cg->m_timesync_transport.length()) {
+        if ((cg->m_timesync_transport.compare("UDP") == 0) ||
+            (cg->m_timesync_transport.compare("1") == 0)) {
+            g_opts->m_timesync_transport = TimesyncTransport::UDP;
+        }
+    }
+
```

Comment:
We could write down the information about transport being used somewhere around line 6487